README.md

```diff
@@ -55,14 +55,14 @@ NOTE: please do *NOT* leave blank lines in the ip_list.txt.
 Third, run the below command to train an RGCN model to perform node classification on the partitioned arxiv graph.
 
 ```
-python3 ~/dgl/tools/launch.py \
+python3 -m graphstorm.run.launch \
         --workspace /tmp/ogbn-arxiv-nc \
-        --num_trainers 1 \
-        --num_servers 1 \
-        --num_samplers 0 \
-        --part_config /tmp/ogbn_arxiv_nc_train_val_1p_4t/ogbn-arxiv.json \
-        --ip_config  /tmp/ogbn-arxiv-nc/ip_list.txt \
-        --ssh_port 2222 \
+        --num-trainers 1 \
+        --num-servers 1 \
+        --num-samplers 0 \
+        --part-config /tmp/ogbn_arxiv_nc_train_val_1p_4t/ogbn-arxiv.json \
+        --ip-config  /tmp/ogbn-arxiv-nc/ip_list.txt \
+        --ssh-port 2222 \
         "python3 /graphstorm/training_scripts/gsgnn_np/gsgnn_np.py \
```

Comment:
In our tutorial, this launch has been changed to the following command. So it is better to change the readme to be consistent with our tutorial.

```
python3  -m graphstorm.run.gs_node_classification \
        --workspace /tmp/ogbn-arxiv-nc \
        --num-trainers 1 \
        --num-servers 1 \
        --num-samplers 0 \
        --part-config /tmp/ogbn_arxiv_nc_1p/ogbn-arxiv.json \
        --ip-config  /tmp/ogbn-arxiv-nc/ip_list.txt \
        --ssh-port 2222 \
        --cf /graphstorm/training_scripts/gsgnn_np/arxiv_nc.yaml \
        --save-model-path /tmp/ogbn-arxiv-nc/models
```